### ISSUE TITLE
Nextcloud 25 support

### ISSUE THREAD

**Issue by kosli** [PanCakeConnaisseur/user_backend_sql_raw#45]
@PanCakeConnaisseur  Have you already looked into supporting Nextcloud 25? I read somewhere that a lot of changes where made on the backend-side with NC 25...maybe this would even help to resolve (or improve) my PR #26 :-)

**Comment by p-bo** [comment]
Just in case it wasn't noticed - Nextcloud improved policy regarding changes in core to ease app developers in maintaining compatibility with newer Nextcloud versions:

https://help.nextcloud.com/t/new-process-for-documenting-core-code-changes-that-affect-app-developers/149828?u=p-bo

**Comment by ivarsg** [comment]
@PanCakeConnaisseur Pressure on this issue is getting higher - [according to this](https://help.nextcloud.com/t/patch-releases-26-0-1-25-0-6-and-last-24-release-24-0-12/160569) NC post:  

`24 is now end-of-life (EOL), meaning 24.0.12 is the last supported version of 24. We therefore strongly recommend upgrading to 25 or, even better, 26`

Are there any updated release of `user_backend_sql_raw` with support to NC 25 or even 26 expected soon? The `user_backend_sql_raw` compatibility is the only reason I am (and probably many more users out there are) still on NC 24.

**Comment by iliesh** [comment]
+1

**Comment by jdsn** [comment]
@PanCakeConnaisseur What are the chances to get this nextcloud app updated soon?
There are several users relying on it, as can be seen in the comments and thumbs up.

Is there anything we can help with? Testing, verifying, ...?

**Comment by OrelEagle** [comment]
If someone has a test setup, they can try to force enable the app and check if everything works (login, create/delete user, change user data...).

Based on Issue #38, this is how it can be done:

> however it works fine after forcefully re-enabling it via ./occ app:enable user_backend_sql_raw --force

Note: I used it myself to upgrade to NC23, but only after others reported it was safe... Unfortunately I don't have any test environment.

**Comment by OrelEagle** [comment]
> > however it works fine after forcefully re-enabling it via ./occ app:enable user_backend_sql_raw --force
> 
> Note: I used it myself to upgrade to NC23, but only after others reported it was safe... Unfortunately I don't have any test environment.

Finally I decided to test in my production server (after making a good backup) and everything works fine.

**Comment by rysson** [comment]
Yes, I've modified supported version a few moths ago.
It works in 25.0.2 like a chram.

**Comment by jdsn** [comment]
This plugin works without errors in Nextcloud Hub 7 (28.0.2).

@PanCakeConnaisseur  Please consider updating this plugin.

Otherwise we'd have to register a new plugin with nextcloud for a fork of this repo.

**Comment by PanCakeConnaisseur** [comment]
There is a high chance that I will update this app in the next few weeks, since I do use it myself. It is just that this is a hobby project for me and thus has a low priority when compared to my day job and other personal things.

**Comment by PanCakeConnaisseur** [comment]
New release with support up to Nextcloud 29 is out.
